x/uniswap/internal/keeper/querier.go

```diff
@@ -0,0 +1,78 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// NewQuerier creates a querier for uniswap REST endpoints
+func NewQuerier(k Keeper) sdk.Querier {
+	return func(ctx sdk.Context, path []string, req abci.RequestQuery) (res []byte, err sdk.Error) {
+		switch path[0] {
+		case types.QueryBalance:
+			return queryBalance(ctx, req, k)
+		case types.QueryLiquidity:
+			return queryLiquidity(ctx, req, k)
+		case types.Parameters:
```

Comment:
I think this is a typo. Should be QueryParameters instead. Similarly, I'm guessing the method to be called is queryParameters()